lib/models/friend.js

```diff
@@ -0,0 +1,26 @@
+const pool = require('../utils/pool');
+
+module.exports = class Friend {
+  id;
+  name;
+  status;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.status = row.status;
+  }
+
+  static async getAll() {
+    const { rows } = await pool.query('SELECT * FROM friends;');
+    return rows.map((row) => new Friend(row));
+  }
+  static async getByID(id) {
+    const { rows } = await pool.query('SELECT * FROM friends WHERE id=$1;', [
+      id,
```

Comment:
Don't forget our filename in our models folder will need to have the first letter capitalized and singular (Friend.js).